plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -290,6 +290,12 @@ public IcebergMetadata(
         this.fileSystemFactory = requireNonNull(fileSystemFactory, "fileSystemFactory is null");
     }
 
+    @Override
+    public boolean schemaExists(ConnectorSession session, String schemaName)
```

Comment:
A follow-up for Delta & Hive would be welcomed:
`io.trino.plugin.deltalake.metastore.HiveMetastoreBackedDeltaLakeMetastore#getDatabase`
`io.trino.plugin.hive.metastore.SemiTransactionalHiveMetastore#getDatabase`

Reply:
discussed offline; it's unclear whether worth it